### PR TITLE
initialize to max value so that there is no conflict when finding old…

### PR DESCRIPTION
**Pull request by heejaechang**
…est cache.

this address https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems/edit/179939

### CODE REVIEW

**Comment by heejaechang** [comment]
@mavasani @srivatsn can you take a look?

File: src/EditorFeatures/Core/Implementation/Workspaces/ProjectCacheService.SimpleMRUCache.cs [hunk 1]
```diff
@@ -37,7 +37,7 @@ public bool Empty
             public void Touch(object instance)
             {
                 var oldIndex = -1;
-                var oldTime = DateTime.UtcNow;
+                var oldTime = DateTime.MaxValue;
```
Comment (mavasani) [inline src/EditorFeatures/Core/Implementation/Workspaces/ProjectCacheService.SimpleMRUCache.cs]: @heejaechang Can you give more context where the IndexOutOfRange was happening? Was `oldIndex = -1` at the end of the loop? If so, shouldn't we have seen an InvalidOperationException from the contract check before we hit IndexOutOfRange? If not, I don't see how `oldIndex` can be greater than `i`.

Comment (heejaechang) [inline src/EditorFeatures/Core/Implementation/Workspaces/ProjectCacheService.SimpleMRUCache.cs]: Require is debug only.

Comment (mavasani) [inline src/EditorFeatures/Core/Implementation/Workspaces/ProjectCacheService.SimpleMRUCache.cs]: Ah ok, then the change LGTM. Thanks.

**Pull request merged**